evm/testv2/SignedSafeMath_test.ts

```diff
@@ -1,71 +1,88 @@
-import * as h from '../src/helpers'
-import { assertBigNum } from '../src/matchers'
-const ConcreteSignedSafeMath = artifacts.require('ConcreteSignedSafeMath.sol')
+import * as h from '../src/helpersV2'
+import { assertBigNum } from '../src/matchersV2'
+import { ethers } from 'ethers'
+import ganache from 'ganache-core'
+import { AbstractContract } from '../src/contract'
+import { createFundedWallet } from '../src/wallet'
 
-contract('SignedSafeMath', () => {
-  let adder, response
-  const INT256_MAX = web3.utils.toBN(
+const ConcreteSignedSafeMathStatic = AbstractContract.fromArtifactName(
+  'ConcreteSignedSafeMath',
+).toStatic()
+const ganacheProvider: any = ganache.provider()
+let defaultAccount: ethers.Wallet
+
+before(async () => {
+  const { wallet } = await createFundedWallet(ganacheProvider, 0)
+  defaultAccount = wallet
+})
+
+describe('SignedSafeMath', () => {
+  // a version of the adder contract where we make all ABI exposed functions constant
+  let adderStatic: ethers.Contract
+
+  let response
+  const INT256_MAX = ethers.utils.bigNumberify(
     '57896044618658097711785492504343953926634992332820282019728792003956564819967',
   )
-  const INT256_MIN = web3.utils.toBN(
+  const INT256_MIN = ethers.utils.bigNumberify(
     '-57896044618658097711785492504343953926634992332820282019728792003956564819968',
   )
 
   beforeEach(async () => {
-    adder = await ConcreteSignedSafeMath.new()
+    adderStatic = await ConcreteSignedSafeMathStatic.deploy(defaultAccount)
```

Comment:
Similar to my question in `contract.ts`, what significance does "static" have? What difference does it make to a developer, or why mark it everywhere?

Reply:
Addressed this above ^